doc/api/http.md

```diff
@@ -3206,6 +3206,41 @@ server.on('request', (request, res) => {
 server.listen(8000);
 ```
 
+## `http.createSimpleServer([directory[, port[, host]]])`
+
+<!--YAML
+added: REPLACEME
+-->
+
+> Stability: 0 - Experimental
+
+* `directory` {string|URL} Root directory from which files would be serve.
+  **Default:** `process.cwd()`.
+* `port` {number}
+* `host` {string} **Default:** `'localhost'`
+
+* Returns: {http.Server}
+
+Start a TCP server listening for connections on the given `port` and `host`, and
+serve statically local files, using `directory` as the root. Please note that
+when specifying a `host` other than `localhost`, you are exposing you local file
+system to all the machines that can connect to your computer.
+
+If `port` is omitted or is 0, the operating system will assign an arbitrary
+unused port, which will be output the standard output.
```

Comment:
I'm unsure what this is trying to say here, it's output will be the standard output or it'll output whatever comes from the standard output?

```suggestion
unused port, which it's output will be the standard output.
```

or whatever that fits correctly here.